src/components/common/container.js

```diff
@@ -1,11 +1,17 @@
 import React from 'react'
 import containerStyles from './container.module.scss'
 
-export default ({ children, narrow }) => (
-  <div
-    className={`container ${containerStyles.container} ${narrow &&
-      containerStyles.narrow}`}
-  >
-    {children}
-  </div>
-)
+export default ({ children, narrow }) => {
+  if (narrow) {
+    return (
+      <div className={`container ${containerStyles.container}`}>
+        <div className={`${narrow && containerStyles.containerNarrow}`}>
```

Comment:
This is within an if statement for narrow = true, so you can remove the `narrow &&` part.

Reply:
Actually, I think you can just use this and not wrap it in an if statement:

```
<div className={`container ${containerStyles.container} ${narrow && conatinerStyles.containerNarrow}`}>
```

Then change the SCSS module to:

```scss
&.container-narrow {. <-- remove the space
```

The extra class will make it more specific than the general container component.

Reply:
Thanks @kevee. I'll remove the redundant `narrow &&` part. 

On your second comment, take a look at the deploy preview (https://deploy-preview-474--upbeat-lovelace-3e9fff.netlify.com/about-project). I made `container-narrow` a child of the primary container so it stays on the page's  grid (keeping the content even with the page title). I may be missing a better solution, but applying the narrow style to the first container div centers it on the page, which is essentially the original change.

Reply:
Ah gotcha, I'm fine with that.